node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/nodeagent/NodeAgentImpl.java

```diff
@@ -601,27 +600,13 @@ private void pushMetricsToContainer(NodeAgentContext context, List<DimensionMetr
             for (DimensionMetrics dimensionMetrics : metrics) {
                 params.append(dimensionMetrics.toSecretAgentReport());
             }
-        } catch (JsonProcessingException e) {
-            // TODO: wrap everything into one try-block (to avoid 'return') when old metrics proxy is discontinued
-            context.log(logger, LogLevel.WARNING, "Failed to wrap metrics in secret agent report", e);
-            return;
-        }
-        String wrappedMetrics = "s:" + params.toString();
-
-        // Push metrics to the metrics proxy in each container
-        runPushMetricsCommand(context, wrappedMetrics, true);
-        runPushMetricsCommand(context, wrappedMetrics, false);
-    }
+            String wrappedMetrics = "s:" + params.toString();
 
-    // TODO: Clean up and inline method when old metrics proxy has been discontinued.
-    private void runPushMetricsCommand(NodeAgentContext context, String wrappedMetrics, boolean newMetricsProxy) {
-        int port = newMetricsProxy ? 19095 : 19091;
-        String[] command = {"vespa-rpc-invoke",  "-t", "2",  "tcp/localhost:" + port,  "setExtraMetrics", wrappedMetrics};
-        try {
+            // Push metrics to the metrics proxy in each container
+            String[] command = {"vespa-rpc-invoke",  "-t", "2",  "tcp/localhost:19095",  "setExtraMetrics", wrappedMetrics};
```

Comment:
What about Vespa 6 nodes?

Reply:
Good point, they would not get the node metrics. Let's put this on hold until stream is on 7.